src/grafana/prometheus/docker/grafana/dashboards/scdf-servers.json

```diff
@@ -0,0 +1,5347 @@
+{
```

Comment:
Is this a generated file from a dashboard configuration?

Reply:
yes